File: mario.c
#include <cs50.h>
#include <stdio.h>

int main()
{
    int height;
    do
    {
        height = get_int("Enter height: ");
    } 
    while (height < 1 || height > 8);

    for(int i=1; i <= height; i++)
    {
        int k = i;
        int ctr=0;
        for(int j=1; j <= height - i; j++)
        {
            printf(" ");
        }
        while(k!=0)
        {
            printf("#");
            --k;
        }
        printf("  ");
        k=i;
        while(k!=0)
        {
            printf("#");
            --k;
        }
        printf("\n");
    }
}

File: pset6/mario.py
import sys
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

for j in range(height, 0, -1):
    for i in range(1, j, 1):
        print(" ", end="") 
    k = j
    while k != height+1:
        print("#", end="")
        k += 1
    print("  ", end="")
    k = j
    while k != height+1:
        print("#", end="")
        k += 1
    print()
